### PR TITLE
Dcip

### PR DESCRIPTION
**Pull request by thast**
update DCIP lab:

-add Cylinder app for fields visualisation
- Remove dependency to SimPEGDCIP and transfer to SimPEG.EM.Static
-deactivate continuous_update

To-Do for next PR:
-still 2D (lines sources)
-still a FudgeFactor for PotentialApp

### CODE REVIEW

**Comment by thast** [comment]
Also, this lab is not currently compatible with DCIP/dev:
Examples:
-SigmaMap is still mapping
- a _get function has changed?

File: DC/Lab10/EOSC350Lab.cls [hunk 1]
```diff
@@ -0,0 +1,153 @@
+%==============================
```
Comment (lheagy) [inline DC/Lab10/EOSC350Lab.cls]: can you please remove this from here: the pdf should only be in the private eosc350 repo and on the eosc350website

File: DC/Lab10/notebook/DCLayers.py [hunk 2]
```diff
@@ -0,0 +1,498 @@
+#import sys
```
Comment (lheagy) [inline DC/Lab10/notebook/DCLayers.py]: can you please move this to the `DC/` folder? (we don't need to have more layers, keeping the tree to 1 level is ideal.)

File: DC/Lab10/notebook/DCWidget.py [hunk 3]
```diff
@@ -0,0 +1,681 @@
+#import sys
```
Comment (lheagy) [inline DC/Lab10/notebook/DCWidget.py]: same comment here re: organization

File: DC/Lab10/EOSC350Lab.cls (hunk 1 above)
Comment (thast) [inline DC/Lab10/EOSC350Lab.cls]: yes I did not moved the .tex and the .pdf but I forgot the .cls (which is the just the layout). I remove it

File: DC/Lab10/notebook/DCLayers.py (hunk 2 above)
Comment (thast) [inline DC/Lab10/notebook/DCLayers.py]: done

File: DC/Lab10/notebook/DCWidget.py (hunk 3 above)
Comment (thast) [inline DC/Lab10/notebook/DCWidget.py]: done

**Pull request merged**